### PR TITLE
userguide: intro doc updates v2

### PR DESCRIPTION
**Pull request by jmtaylor90**
Make sure these boxes are signed before submitting your Pull Request -- thank you.

- [x] I have read the contributing guide lines at https://redmine.openinfosecfoundation.org/projects/suricata/wiki/Contributing
- [x] I have signed the Open Information Security Foundation contribution agreement at https://suricata.io/about/contribution-agreement/
- [x] I have updated the user guide (in doc/userguide/) to reflect the changes made (if applicable)

Link to [redmine](https://redmine.openinfosecfoundation.org/projects/suricata/issues) ticket:
https://redmine.openinfosecfoundation.org/issues/5385

Describe changes:
- updated with changes requested in https://github.com/OISF/suricata/pull/7492
-
-

#suricata-verify-pr:
#suricata-verify-repo:
#suricata-verify-branch:
#suricata-update-pr:
#suricata-update-repo:
#suricata-update-branch:
#libhtp-pr:
#libhtp-repo:
#libhtp-branch:

### CODE REVIEW

File: doc/userguide/rules/intro.rst [hunk 1]
```diff
@@ -6,12 +6,16 @@ people are using existing rulesets.
 
 The official way to install rulesets is described in :doc:`../rule-management/suricata-update`.
 
+There are a number of free rulesets that can be used via suricata-update.
+To aid in learning about writing rules, the Emerging Threats Open (free) is
```
Comment (jlucovsky) [inline doc/userguide/rules/intro.rst:10]: nit:s/Open/Open ruleset/

File: doc/userguide/rules/intro.rst [hunk 2]
```diff
@@ -73,6 +74,11 @@ concerns. You can choose between four basic protocols:
 * icmp
 * ip (ip stands for 'all' or 'any')
 
+There are a couple of additional tcp related protocol options:
```
Comment (jlucovsky) [inline doc/userguide/rules/intro.rst:77]: s/tcp/TCP/

File: doc/userguide/rules/intro.rst [hunk 3]
```diff
@@ -100,21 +106,21 @@ you can pick from. These are:
...
 
-The availability of these protocols depends on whether the protocol is enabled in the configuration file suricata.yaml.
+The availability of these protocols depends on whether the protocol
+is enabled in the configuration file, suricata.yaml.
 
-If you have a signature with for
-instance a http protocol, Suricata makes sure the signature can only
-match if it concerns http-traffic.
+If you have a signature with the protocol declared as 'http', Suricata makes
+sure the signature will only match if the tcp stream contains http traffic.
```
Comment (jlucovsky) [inline doc/userguide/rules/intro.rst:113]: s/tcp/TCP

**Comment by jmtaylor90** [comment]
Thanks @jlucovsky! Continued in https://github.com/OISF/suricata/pull/7554

**Pull request closed without merging**